readthedocs/projects/tasks.py

```diff
@@ -621,6 +621,11 @@ def finish_build(version_pk, build_pk, hostname=None, html=False,
         version.built = True
         version.save()
 
+    if not pdf:
```

Comment:
We need the `readthedocs.yml` config soon :( this would solve that problem as well ...